README_PAYLOAD.md

```diff
@@ -61,21 +61,26 @@ todo
 ```
 Version is: 0001|0011 => 1.3 (signed message with chained signatures)  https://github.com/ubirch/ubirch-protocol/blob/master/README.md#field-types
 Type is: 0x54 (84)  https://github.com/ubirch/ubirch-protocol/blob/master/README.md#payload-type
-Example Payload:
-```
- [ "v1.0.2-PROD-20180326103205 (v5.6.6)",
-  6496,
-  3,
-  {
-    1594155266: 3665,
-    1594155326: 3682
-   },
-   {
-    "min": 3500,
-    "max": 4200,
-    "i": 60000,
-    "il": 1800000
-   }
+
+trackle message payload:
+```
+  [ // <msgpack.array[5]
+    "v1.0.2-PROD-20180326103205 (v5.6.6)", // <msgpack.raw> Version String
+    5, // <msgpack.uint32> wakeup cycles
```

Comment:
is there some description about what exactly a wakeup count will trigger?

Reply:
the wakeup cycle counter is triggered once every measuring interval. the interval duration is part of the configuration map. it is the "i" value in milliseconds